scipy/ndimage/tests/test_filters.py

```diff
@@ -750,9 +770,13 @@ def test_gauss_axes_kwargs(self, axes):
         assert_allclose(output, expected)
 
     @pytest.mark.parametrize(
-        'axes', [(1.5,), (0, 1, 2, 3), (3,), (-4,)]
-    )
-    def test_gauss_invalid_axes(self, axes):
+        'filter_func, kwargs',
+        [(ndimage.gaussian_filter, {'sigma': 1.0}),
```

Comment:
It looks like this approach - including kwargs in the parameterized cases - would simplify `test_filter_axes` considerably. Why not do the same there?